common/metrics/statsdreporter.go

```diff
@@ -127,14 +127,26 @@ func mergeDicts(dic1 map[string]string, dic2 map[string]string) (resultDict map[
 	return
 }
 
+// Cherami's metrics are tagged, but here we need to flatten 
+// it for statsd. We order tags by: Operation, Destination, 
+// ConsumerGroup, Hostname
+var tagsFlattenOrder = []string{
```

Comment:
is this the right place for defining the possible tags and the order in which they appear ? 
What happens when someone introduces a new tag in metrics/defs.go ?